src/typescript/sdk/src/server/env.ts

```diff
@@ -1,7 +1,3 @@
-if (process.env.NODE_ENV !== "test") {
-  require("server-only");
-}
-
```

Comment:
We need to leave this in so we avoid importing any of these values into the client, otherwise we could accidentally print them our or something.

I imagine this gave you issues, I'll try to fix it today but if I don't get around to it we need to make sure this check exists in some form

Reply:
Fixed in a similar way I fixed the jest tests